lib/src/connector/connection.cc

```diff
@@ -492,11 +493,16 @@ WS_Context_Ptr Connection::onTlsInit(WS_Connection_Handle hdl)
 void Connection::onClose(WS_Connection_Handle hdl)
 {
     Util::lock_guard<Util::mutex> the_lock { state_mutex_ };
-    timings.close = Util::chrono::high_resolution_clock::now();
+    timings.setClosed();
     auto con = endpoint_->get_con_from_hdl(hdl);
     LOG_DEBUG("WebSocket on close event: {1} (code: {2}) - {3}",
               con->get_ec().message(), con->get_remote_close_code(),
               timings.toString());
+
```

Comment:
I'll make a change to the above DEBUG message and avoid reporting the error and code if the code is 1000; will say that it's normal in such case.